src/recoilState/merkleDistributor.ts

```diff
@@ -0,0 +1,22 @@
+import {
```

Comment:
Don't have separate files for these in `recoilState` just put it in appState for now, later I might reorganize them